x-pack/plugins/enterprise_search/server/utils/create_ml_inference_pipeline.ts

```diff
@@ -22,9 +22,9 @@ import {
  * Details of a created pipeline.
  */
 export interface CreatedPipeline {
-  id: string;
-  created?: boolean;
```

Comment:
These were just sort orders that I fixed and honestly don't belong in this PR... my apologies.

Reply:
Vice versa, fix them wherever you can, unless it would cause a lot of trouble.